python/test/t_LinearLeastSquares_std.py

```diff
@@ -34,14 +34,6 @@
 point[0] = 0.25
 point[1] = 0.25
 data[8] = point
-myLeastSquares = ot.LinearLeastSquares(data, myFunc)
-myLeastSquares.run()
-responseSurface = ot.Function(myLeastSquares.getMetaModel())
-print("myLeastSquares=", repr(myLeastSquares))
-print("responseSurface=", repr(responseSurface))
-inPoint = ot.Point(myFunc.getInputDimension(), 0.1)
-print("myFunc(", repr(inPoint), ")=", repr(myFunc(inPoint)))
-print("responseSurface(", repr(inPoint), ")=", repr(responseSurface(inPoint)))
```

Comment:
Removing this part of the unit test does not seem to be necessary according to me because the `repr` will go untested then. Changing the code to:
```
myLeastSquares = ot.LinearLeastSquares(data, myFunc(data))
```
and removing the evaluation of `getMetaModel` is sufficient.

Reply:
it would duplicate the other part of the test